roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -48,14 +48,20 @@ def run(self, tmp, task_vars):
         if failed:
             if "Error connecting: Error while fetching server API version" in msg:
                 msg = (
-                    "It appears Docker is not running.\n"
+                    "It appears Docker is not running. This check will be skipped.\n"
                     "Please start Docker on this host before running this check.\n"
                     "The full error reported was:\n  " + msg
                     )
+
+                return {
+                    "failed": False,
+                    "changed": changed,
+                    "msg": msg,
+                }
             elif "Failed to import docker-py" in msg:
                 msg = (
                     "The required Python docker-py module is not installed.\n"
-                    "Suggestion: install the python-docker-py package on this host."
+                    "Suggestion: install the python docker-py package on this host."
```

Comment:
Add least in Fedora the RPM package is indeed called `python-docker-py`.

If you don't mean the RPM package name, then _Python_ should be capitalized.